x-pack/plugins/security_solution/common/endpoint/types.ts

```diff
@@ -433,11 +433,26 @@ export interface EndpointEvent {
   process: {
     entity_id: string;
     name: string;
+    executable: string;
+    args: string;
+    code_signature: {
+      status: string;
+      subject_name: string;
+    };
+    pid: number;
+    hash: {
+      md5: string;
+    };
     parent?: {
       entity_id: string;
       name?: string;
+      pid: number;
     };
   };
+  user: {
+    domain: string;
```

Comment:
I think we'll want this as optional because I don't think it'll be set for linux.

Reply:
Good spot, thanks Jonny. https://github.com/elastic/kibana/pull/69098/commits/0548b071933b106d4b259e912218f58372c6fcd1